commands/files/readerfile.go

```diff
@@ -13,10 +13,12 @@ type ReaderFile struct {
 	fullpath string
 	reader   io.ReadCloser
 	stat     os.FileInfo
+	offset   int64
+	baseInfo ExtraInfo
 }
 
-func NewReaderFile(filename, path string, reader io.ReadCloser, stat os.FileInfo) *ReaderFile {
-	return &ReaderFile{filename, path, reader, stat}
+func NewReaderFile(filename, path, abspath string, reader io.ReadCloser, stat os.FileInfo) *ReaderFile {
```

Comment:
this can probably be changed so that it only accepts the absolute path instead of a filename, path and absolute path.